cos-fleetshard-sync/src/main/java/org/bf2/cos/fleetshard/sync/FleetShardSyncConfig.java

```diff
@@ -157,6 +157,15 @@ interface Resources {
         @WithDefault("30s")
         @WithConverter(DurationConverter.class)
         Duration housekeeperInterval();
+
+        /**
+         * Determine how often the cleaning of old metrics should happen.
+         *
+         * @return the metricshousekeeping interval
+         */
+        @WithDefault("24h")
+        @WithConverter(DurationConverter.class)
+        Duration metricsHousekeeperInterval();
```

Comment:
just as a note: at some point, we may think about adding a generic `daily` housekeeping runner